msteams-platform/apps-in-teams-meetings/teams-together-mode.md

```diff
@@ -0,0 +1,239 @@
+---
+title: Together Mode in Teams
+description: Work with Together Mode 
+ms.topic: conceptual
+---
+
+# Together Mode in Teams
+
+Teams Together Mode provides a simple, innovative, and user-friendly meeting environment. It digitally combines participants into a single virtual space instead of being shown separately in boxes. It creates your virtual image and places the image in a virtual environment where you can easily interact with your peers. It makes meetings more engaging in the auditorium view.
+
+> [!VIDEO https://www.youtube-nocookie.com/embed/MGsNmYKgeTA]
+
+You can create scene only apps or extend your current app to have scenes for Together Mode. A scene is a virtual view of the auditorium or meeting venue where the attendees are placed and includes the seats for attendees.
+
```

Comment:
A scene is more than auditorium. It can be anything that the scene developer wants. Lets reword this: 

**Scenes** in Together Mode is an artifact created by the scene developer using the _scene design studio_  (we will need another page where we will explain an end to end scene building exercise in scene design studio) that brings together people along-with their video stream in a creative setting as conceived by the scene creator. A scene brings people together by having designated **seats** in the conceived scene setting and participant video streams are rendered in those seats.

Reply:
Please provide the details as per this comment "we will need another page where we will explain an end to end scene building exercise in scene design studio"